README.md

```diff
@@ -167,6 +167,6 @@ I would like to thank Microverse Foundation for their amazing templates.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](/.LICENSE/MIT.md) licensed.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Great job adding and customizing README. However, there are just few things that need to be fixed and I know you can pull this off. Right ?🎯

- [ ] I think your README logo is broke. When I'm clicking on it, I'm getting an error. No worries let's fix this now ✌

![image](https://user-images.githubusercontent.com/106119105/213694874-59033c1e-1af4-4e16-9090-df1e0b18018d.png)
![image](https://user-images.githubusercontent.com/106119105/213694907-7bfba7dd-5fed-4d87-9fc8-69090b61c4dc.png)

- [ ] I can see that you have edited the other sections very well. Good job 👍 I think it would be better if you remove the description/placeholder that I've marked below 🔽 These placeholders are there for your guidance of wat to add in the respective section. Let's fix this thing 💪
![image](https://user-images.githubusercontent.com/106119105/213695417-abf8b99f-ddae-4239-9a76-d59704d06b86.png)

- [ ] Just one more thing to go. You are almost there ✈ Great job adding the section here. However, I think it would be better if you follow te README template and edit it as given in README [template] ()

## **_Your README 👇_**
![image](https://user-images.githubusercontent.com/106119105/213695902-45ab0102-7e04-4155-ac9e-51b029209f63.png)

## README template 👇
![image](https://user-images.githubusercontent.com/106119105/213695975-2c37dc09-ee2c-4a32-acfc-1d2969d4df79.png)


**_Note: Please note that this applies to your entire README._**